discovery-kubernetes-api/src/main/resources/reference.conf

```diff
@@ -0,0 +1,30 @@
+######################################################
+# Akka Service Discovery DNS Config                  #
+######################################################
+
+akka.discovery {
+
+  # Set the following in your application.conf if you want to use this discovery mechanism:
+  # impl = akka-dns
```

Comment:
`# method = kubernetes-api` AFAIR, this is so people can copy paste this to enable a given impl

Reply:
Also fixed